docs-gen/content/rule_set/basics.md

```diff
@@ -103,22 +103,37 @@ Vehicle.Navigation.CurrentLocation:
 
 It is recommended for servers, which are implementing protocols for the vehicle signal specification, to serve old and new nodes during the deprecation period described above.
 
+## Naming Rules
+
+Different languages and environments have different rules for identifiers.
+To simplify usage of VSS in as many languages and environments as possible the following rules apply.
+
+* The allowed character set for node names (branches and signals) is `A-Z`, `a-z` and `0-9`
+* Node names must not start with a digit
+* Names are case sensitive, but it is not allowed to have two nodes that only differ by case,
+  i.e. it is not allowed to have both `Vehicle.Aaa` and `Vehicle.AaA`
+* It is not allowed to reuse the same node name as a parent node in a child node,
+  i.e. it is not allowed to have a node `A` if a parent grand parent also is called `A`, like `A.B.A`
+* For string literals only the characters `A-Z`, `0-9` and `_` are allowed,
+  and the value must start with `A-Z`. See the [chapter](/vehicle_signal_specification/rule_set/data_entry/allowed/) for specifying allowed values.
+
+It is recommended that VSS Tools shall give an error if the rules above are not followed.
+
 ## Style Guide
 
-The VSS specification must adhere to YAML syntax. To keep the standardized VSS specification in this repository consistent the following style guide is provided.
+For a contribution to VSS standard catalog the following recommendations must be followed:
 
 ### Naming Conventions
 
-The recommended naming convention for node elements is to use camel case notation starting with a capital letter. It is recommended to use only
-`A-Z`, `a-z` and `0-9` in node names. For boolean signals it is recommended to start the name with `Is`.
+The recommended naming convention for node elements is to use camel case notation starting with a capital letter.
```

Comment:
As well, it seems to be common practice that acronyms are in uppercase in node names (which, by the way, makes it a bit awkward to read when the acronym is in the middle of a node name).

Given that acronyms are indeed "words", my personal preference is for acronyms in code, etc. to follow the same case use as codes. For example, Vehicle.Adas.Abs.IsEnabled.

Reply:
Adding that to the PR to get comments

Reply:
Personally, in my eyes, reading "Adas" or "Abs" looks weird as opposed to "ADAS" or "ABS". In case it turns out I am in the minority here, I can live with it :)